src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+import logging
+from typing import List
+
+from sentry.integrations.github.client import GitHubAppsClient
+from sentry.models.pullrequest import PullRequest
+from sentry.models.repository import Repository
+from sentry.shared_integrations.exceptions.base import ApiError
+from sentry.tasks.integrations.github.pr_comment import RATE_LIMITED_MESSAGE, GithubAPIErrorType
+from sentry.utils import metrics
+
+logger = logging.getLogger(__name__)
+
+OPEN_PR_METRIC_BASE = "github_open_pr_comment.{key}"
+
+# Caps the number of files that can be modified in a PR to leave a comment
+OPEN_PR_MAX_FILES_CHANGED = 7
```

Comment:
are these arbitrary?

Reply:
they can be changed

Reply:
but in a sense they are arbitrary. it's just that the more files that are changed in a PR the less likely finding issues associated with those files will actually be useful